### PR TITLE
Feature/daylight saving callout

### PR DESCRIPTION
**Pull request by simonschwartz**
# Reminder callout
## Description

This PR adds styling and documentation for a reminder callout. The reminder callout is currently used on the times and dates section of gov.au
## Additional information

reminder-callout

### CODE REVIEW

File: assets/sass/_typography.scss [hunk 1]
```diff
@@ -537,6 +539,58 @@ Style guide: Typography.6 Callouts & warnings
...
+  }
+
+  h2,
+  h3,
+  h4 {
+    margin-top: $tiny-spacing;
+  }
+
+  &:before {
+    background-image: url('https://cdn.rawgit.com/AusDTO/gov-au-images/master/savings-forward.png');
```
Comment (maxious) [inline assets/sass/_typography.scss]: this should be imported into ui kit... and probably as a SVG?

File: assets/sass/_typography.scss [hunk 2]
```diff
@@ -537,6 +539,58 @@ Style guide: Typography.6 Callouts & warnings
...
+      height: 2 * $base-spacing;
+      left: $medium-spacing;
+      position: absolute;
+      top: 1.5 * $base-spacing;
+      width: 2 * $base-spacing;
+    }
+  }
+
+  &.icon-back:before {
+    background-image: url('https://cdn.rawgit.com/AusDTO/gov-au-images/master/savings-back.png');
```
Comment (maxious) [inline assets/sass/_typography.scss]: ditto, merge into repo

**Comment by klepas** [comment]
Mhh. I think I remember saying to @simonschwartz that eh, let’s be lazy, but you’re probably right @maxious. @petronbot — thoughts?

I can also see some overlap styles and user needs between:
- calendar table styles (`.calendar-table`),
- `.callout--reminder` and
- `.callout-calendar-event`

@petronbot what do you think about making `.callout--reminder` a subtype of `.callout-calendar-event` (eg `.callout--calendar-event__reminder` or `.callout--event__reminder`) or something like that?

**Comment by klepas** [comment]
Also, the sub-styles of these two classes have potential to share the same styling: is there a reason the text sizes differ ever so slightly between the two types? (came up in a chat between Simon and myself)

**Comment by hannah-ustwo** [comment]
[STATUS UPDATE - Addressing soon] This was deprioritized for the Aug 4th demo and was put on hold for UI-kit. With newly-established priorities to remove custom CSS from platform for August 30th, this will be addressed by UI-kit in one of the upcoming 3 sprints leading up to private beta. (woot)

**Comment by hannah-ustwo** [comment]
Correcting previous statement - With testing and debugging for cross-browser compatibility being TOP priority for Aug 30, if this has much further work to be done ( @klepas @petronbot ?) , it will likely be addressed by UI-kit post sprint 34.

**Comment by klepas** [comment]
@hannah-ustwo This falls into the category of callouts, which are being revisited (can’t remember the card ID) to ensure the design is unified.

**Comment by hannah-ustwo** [comment]
@klepas I've kept it separate for now - I had a chat with Gary and he reckons that if there is a PR that is good to go, we can take it in. The design polish over whole call-out system can happen later down the line and shouldn't block this for now.

**Comment by klepas** [comment]
@hannah-ustwo If we’re going to potentially overhaul _the same section of code_ I’d pref. to do them together.

**Comment by hannah-ustwo** [comment]
Fair enough @klepas. Extra work for something disposable is sh!t.

**Comment by maxious** [comment]
https://github.com/AusDTO/gov-au-beta/blob/develop/app/assets/stylesheets/_daylight-savings-callout.scss is the latest version of this if it is too hard to merge now

**Comment by klepas** [comment]
@simonschwartz @maxious Hey, just revisiting this. I’m inclined to close this PR and open a new one based on the stylesheet Alex linked above.

I guess a related discussion is: do we still want to include all the current features sitting inside Sites that are missing from uikit?

**Comment by klepas** [comment]
Closing.

**Pull request closed without merging**